client/js/audio.js

```diff
@@ -33,36 +33,58 @@ async function loadAudioBuffer(audioSource) {
   }
 }
 
-async function addAudio(audioSource, maxVolume, length) {
+async function addAudio(audioSource, maxVolume, length, count = 1) {
```

Comment:
Is setting `count = 1` as a default needed here given that we already have a way for handling defaults in the JSON?  In other words, is there any other way addAudio is being called, where the default count wouldn't already be 1?

Reply:
Yes.  Otherwise clickSound does not work.  I tried a couple of different ways and this seemed the best solution.

Reply:
> we might even be able to get rid of sending maxVolume, length, and players to the server from clickSound using your method.

I'll give that a try.  I think that would be the simplest, but I don't know "better" either.

Reply:
In the end, I didn't like the way it looked trying it the alternative way.  We sill had to include audioSource and players defined in widget.js, so I just moved count there.  Now everything looks consistent.